joomla/joomla-cms
# [4.0] Remove element selector that does not exist

### Summary of Changes
Closing the batch modal in Tags causes a JavaScript error due to trying to access a non-existing element. Looks like a case of copy/paste because there is of course no Tag selector in the batch modal on the Tags page.


### Testing Instructions
1. Open the developer Console with F12
2. Open the console tab in the developer console
<img width="367" alt="image" src="https://user-images.githubusercontent.com/359377/58588809-ce3baf00-8260-11e9-810c-5ede40b9f15e.png">

3. Go to Components -> Tags
4. Check 1 or more tags (if you have none, create one)
5. Click on the Batch button
6. Click on the Cancel button
7. Notice in the console a JavaScript error
<img width="478" alt="image" src="https://user-images.githubusercontent.com/359377/58588862-ef040480-8260-11e9-98d8-165fbe5e0a57.png">

8. Apply patch
9. Reload page
10. Click on the Batch button
11. Click on the Cancel button
12. Notice in the console there is no JavaScript error


### Expected result
No JavaScript error

### Actual result
A JavaScript error


### Documentation Changes Required
None
